content/docs/import/import-data-assistant.md

```diff
@@ -1,106 +1,100 @@
+</Admonition>
 
-## Create a Neon project
+<Steps>
 
-<div style={{ display: 'flex', alignItems: 'top' }}>
-  <div style={{ flex: '0 0 55%', paddingRight: '20px' }}>
-    By default, we try to create your new project to match your source database:
+## Check Compatibility
 
-    - Matching Postgres **version**
-    - Matching **region**
+Enter your database connection string and we'll verify:
 
-      <Admonition type="note">
-      This is an early feature and may not work for all regions or providers.
-      </Admonition>
+- Database size is within the current 10GB limit
+- Postgres version compatibility (Postgres 14 to 17)
+- Extension compatibility
+- Region availability
 
-    You can modify any of these settings to suit the needs of your database, such as the host region, autoscaling range, and so on.
+## Import Your Data
 
-    See [Create a project](/docs/manage/projects#create-a-project) for more details about these options.
+Once checks pass, we'll:
 
-  </div>
-  <div style={{ flex: '0 0 45%', margin: '-15px 0' }}>
-    ![create Neon project](/docs/import/migration_create_project.png)
-  </div>
-</div>
+- Create a new branch for your imported data.
+- Copy your data automatically using `pg_dump` and `pg_restore`.
+- Verify that the import completed successfully.
 
-## Move data to Neon
+<Admonition type="note">
+During import, your source database remains untouched — we only read from it to create a copy in Neon.
+</Admonition>
 
-Next, we'll send you to the command line. We generate the `pg_dump` and `pg_store` commands, pre-populated with the correct connection strings and required parameters.
+## Next Steps
 
-<div style={{ display: 'flex', justifyContent: 'center'}}>
-  <img src="/docs/import/migration_move_data.png" alt="move data to Neon using pg_dump and pg_restore" style={{ width: '80%', maxWidth: '600px', height: 'auto' }} />
-</div>
+After a successful import:
 
-The `pg_dump` command is populated with your source database:
+1. Find your newly imported database branch on the **Branches** page of your project.
 
-```bash shouldWrap
-pg_dump -Fc -v -d "postgresql://<username>:<password>@<source_host>:<source_port>/<source_database>" -f database.bak
-```
+   ![Branches page showing imported branch](/docs/import/import_data_assistant_branch.png)
 
-The `pg_restore` command uses the connection string for the database in your newly created project in Neon:
+   _Imported branches are typically named with a timestamp, as shown here._
 
-```bash shouldWrap
-pg_restore -v -d "postgresql://<username>:<password>@<destination_host>:<destination_port>/<destination_database>" database.bak
-```
+2. Run some test queries to ensure everything imported correctly.
+3. Switch your connection string to point to your new Neon database.
 
-For more detailed instructions about using these commands, see [Migrate data using pg_dump and pg_restore](/docs/import/migrate-from-postgres).
+</Steps>
 
-## Next Steps
+## Known Limitations
```

Comment:
nit: maybe it is worth moving it as `###` under _Import Your Data_ section? So you quickly see limitations, not looking around